### DISCUSSION TITLE
Code block and/or div fencing weirdness

### DISCUSSION THREAD

**Discussion by grasshoppermouse** [posit-dev/positron#4467] [Q&A] [answered]
The code block at line 665 is not recognized by positron, but when I add text at line 651, it is. I went through my file to confirm I opened and closed all code blocks and div fences, but maybe I missed something. What have I done wrong?

https://github.com/user-attachments/assets/33603a70-76cf-4036-ab07-105c36ad03ff

The entire qmd file is here: https://github.com/grasshoppermouse/strengthmating/blob/main/paper.qmd

**Comment by juliasilge** [comment, marked as answer]
I think you have found a bug in the Quarto extension! I opened this issue: https://github.com/quarto-dev/quarto/issues/521

In the meantime, from my experience trying to simplify your document, I think almost any change you make in the document above this line (like adding a new line inside a div) will get you going again.

**Reply by grasshoppermouse** [reply]
Thanks for your efforts! I discovered this when I noticed that the outline menu on the left wasn't picking up any subheadings below 672.

Relatedly, it would be useful if positron shaded or otherwise demarcated div blocks, as it does for code blocks.
